CHANGES.md

```diff
@@ -62,6 +62,9 @@ Kernel
   since guessing the decreasing argument can now be more expensive.
   (PR #9602)
 
+- Term (constr) comparison ignores the internal boolean of primitive
+  projections.
```

Comment:
Maybe also mention micromega being fixed here, tactic changes can be fragile.

Reply:
This entry is impossible to understand for non-developers. Try to reexpress it in terms of unfolding of the projections, at least.

Reply:
Do non-developers understand the unfolding of primitive projections? Running `unfold p` where `p` is a primitive projections has no visible effect (beyond breaking a few tactics…).